class-debug-bar-cron.php

```diff
@@ -380,6 +381,17 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by scheldule interval
+		 *
+		 * @param array $a first element of compasion  pair
+		 * @param array $b second element of compasion  pair
+		 *
+		 * @return int 1 if element $a interval greater then element $b interval, -1  otherwise.
+		 */
+		function schedules_sorting($a, $b){
+			return (int) $a['interval'] > $b['interval'];
```

Comment:
Sorry once again. Yeah logically it will return true || false, but not 1 and -1. Thanks years of using shortcuts in PHP.  Its corrected now.

As for question how it will affect intervals of other vendors... It wouldn't "in general". I have add few intervals for for testing proposes.
![what_is_the_more_then_one_interval_of_same_ammount](https://cloud.githubusercontent.com/assets/651824/11661321/a1b2567c-9dd1-11e5-9add-814ecde24641.png)